Mathport/Syntax/Translate/Tactic/Lean3.lean

```diff
@@ -9,7 +9,7 @@ open Lean
 open Lean.Elab.Tactic (Location)
 
 namespace Mathport.Translate.Tactic
-open AST3 Parser
+open AST3 Mathport.Translate.Parser
```

Comment:
Could you explain what this does? I saw that there is a new namespace resolution algorithm but I'm not sure why it doesn't work as is here.

Reply:
The new one also opens `Lean.Parser` and then you get the wrong `Lean.Parser.mkIdent` below.